src/gtk/textctrl.cpp

```diff
@@ -34,6 +34,16 @@
 #include "wx/gtk/private/gtk2-compat.h"
 #include "wx/gtk/private/gtk3-compat.h"
 
+#if wxUSE_SPELLCHECK
+extern "C" {
+#if GTK_CHECK_VERSION(3,0,0)
+#include <gtkspell-3.0/gtkspell/gtkspell.h>
+#else // GTK_CHECK_VERSION(3,0,0)
+#include <gtkspell-2.0/gtkspell/gtkspell.h>
```

Comment:
does this also work for gtk1?

Reply:
Good point, I didn't realise that gtk1 is still supported. I wouldn't plan to support gtk1 unless it either a) works by default or b) requires only minimal changes.